product_docs/docs/eprs/7.0/07_common_operations/11_using_ssl_connections.mdx

```diff
@@ -168,6 +168,38 @@ sslusers        enterprisedb            enterprisedb
 
 After you have configured SSL on the Postgres database server, the following steps provide an example of generating a certificate and keystore file for the publication server and subscription server (the JDBC clients).
 
+Before you begin, configure the client for SSL with trigger mode.
+
+- If you are using PostgreSQL, on the SSL-enabled Postgres database server:
+
+  Register the following client/cert files on the publication/subscription database using an SSL connection:
+  - `postgresql.crt`
+  - `postgresql.pk8`
+  - `root.crt`
+
+  In our example, we use the copy of this self-signed certificate and key generated for the database server on the client side.
+
+  The default location of these files is `{user.home}/.postgresql(e.g/var/lib/pgsql/.postgresql/)`. The file location can be overridden using SSL connection parameters or Postgres SSL environmental variables, see [Setting Non-default Paths using Environment Variables](setting-non-default-paths-using-environment-variables) for more information.
+
+  Copy and rename the files:
+  ```
+  $ cd /var/lib/pgsql/.postgresql/
+
+  $ cp /var/lib/pgsql/11/data/server.crt postgresql.crt
+
+  $ cp /var/lib/pgsql/11/data/root.crt .
+
+  $ cp /var/lib/pgsql/11/data/server.key postgresql.key
+
+  $ openssl pkcs8 -topk8 -inform PEM -in postgresql.key -outform DER -out postgresql.pk8 -v1 PBE-MD5-DES -nocrypt
+  ```
+
+- If you are using EDB Postgres Advanced Server, on the SSL-enabled Postgres database server:
+
+  Register the following client/cert files on the publication/subscription database using an SSL connection:
```

Comment:
"Register" is not the correct word to use here, because we wont be registering these certificates in publication/subscription server, we only need these certificates when making connection to publication/subscription database using SSL connection.